.github/workflows/actions.yml

```diff
@@ -271,6 +271,7 @@ jobs:
     - name: Run Tests
       run: |
         $RUNNER --engine="$GITHUB_WORKSPACE/out/extended/walrus" wasm-test-extended
+        $RUNNER --jit --engine="$GITHUB_WORKSPACE/out/extended/walrus" wasm-test-extended
```

Comment:
After this patch, does the extension director work with jit?

Reply:
What do you mean by extension director?